Source/Core/DolphinWX/LogWindow.cpp

```diff
@@ -359,5 +361,13 @@ void CLogWindow::Log(LogTypes::LOG_LEVELS level, const char* text)
   if (msgQueue.size() >= MSGQUEUE_MAX_SIZE)
     msgQueue.pop();
 
-  msgQueue.emplace(static_cast<u8>(level), StrToWxStr(text));
+  // Convert from UTF-8
+  wxString encoded_message = StrToWxStr(text);
+  // Fallback to OS locale if conversion failed
+  if (encoded_message.empty())
+  {
+    PanicAlert(StringFromFormat("Conversion from UTF-8 failed with: \"%s\"", text).c_str());
```

Comment:
PanicAlert calls wxMsgAlert which calls StrToWxStr, so there isn't going to be any text in this panic alert.